### PR TITLE
rishabh submission

### PR DESCRIPTION
**Pull request by rishabh-ranjan**
__final val loss:__ 3.64
__link to val loss:__ https://api.wandb.ai/links/rishabh-ranjan/amq3n70d
__link to train loss:__ https://api.wandb.ai/links/rishabh-ranjan/wlj3umjy
__what i did:__ base model + weight tying + SwiGLU + rotary embedding + compile. model hparams from handout, no dropout, lr 2e-3. mainly made things efficient, so that i can squeeze out max iterations/tokens in the 1.5 hours.

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
